### DISCUSSION TITLE
I accidentally tampered the access rights of my instructor

### DISCUSSION THREAD

**Discussion by tanweiming00** [community/community#50022] [GitHub Education]
Hi, I accidentally tampered the access rights of my instructor to read-only. Everyone is locked out of the repository/team. Is there a way to revert manually? please help im panicking.

**Comment by Anmol-Baranwal** [comment]
If you accidentally changed the access rights of your instructor to read-only and as a result everyone is locked out of the repository or team, there are a few things you can do to try and revert the changes.

1. Ask the affected users to clone the repository: If the repository is public or if the users have access to it, they can clone the repository to their local machine using git clone. This will create a local copy of the repository that they can work with.

2. Revert the change: If you have administrative access to the repository or team, you can try to revert the change that you made. To do this, go to the settings page for the repository or team and look for the access control settings. If you changed the access control settings for the repository or team, you should be able to find an option to revert the change.

3. Contact GitHub support: If you are unable to revert the change yourself or if you need further assistance, you can contact GitHub support for help. They can help you recover access to the repository or team.

It's important to be careful when making changes to access control settings, especially if it affects multiple users. Make sure to double-check the settings before saving any changes, and consider testing changes in a staging environment before applying them to production.

**Reply by Anmol-Baranwal** [reply]
@tanweiming00 
I just wanted to check in and see if you had any further questions or concerns about my answer on GitHub. If you have no further doubts or if your doubts have been resolved, would you mind marking my answer as accepted?

It would be greatly appreciated and it will also help other users who might be looking for a solution to a similar issue.
